js/components/Image.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from "react";
+
+export default class Image extends Component {
+  render() {
+    let { mode, src, height, width, style, ...props } = this.props;
+    let modes = {
+      fill: "cover",
+      fit: "contain"
+    };
+    let size = modes[mode] || "contain";
+
+    let defaults = {
+      height: height || 100,
+      width: width || 100,
+      backgroundColor: "gray"
+    };
+
+    let important = {
```

Comment:
What's the reason for naming this variable "important"? Is there a more semantic name?

Reply:
That managed to sneak in from the reference implementation. No idea why they used that name. I've changed it to "background".